app/src/Application/DbSettingsObj.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <StatusGo/SettingsAPI>
+
+#include <QtCore/QtCore>
+
+namespace Status::Application {
+
+    class DbSettingsObj: public QObject
+    {
+        Q_OBJECT
+
+        Q_PROPERTY(QString address READ address NOTIFY addressChanged)
+        Q_PROPERTY(QString displayName READ displayName NOTIFY displayNameChanged)
+        Q_PROPERTY(QString preferredName READ preferredName NOTIFY preferredNameChanged)
+        Q_PROPERTY(QString keyUid READ keyUid NOTIFY keyUidChanged)
+        Q_PROPERTY(QString publicKey READ publicKey NOTIFY publicKeyChanged)
+
+
+    public:
+        explicit DbSettingsObj(StatusGo::Settings::SettingsDto rawData);
+
+        [[nodiscard]] QString address() const;
```

Comment:
@stefandunca in practice we should not call those functions without using the returned value, but it may happen somebody to do so, I usually was adding them to all item getters and to functions of controller where not using returned value produces some leak or is an error. But as you wish, we can discuss on this.